src/commands/build.ts

```diff
@@ -85,36 +83,58 @@ export class BuildCommand implements Command {
     },
   ];
 
-  run(options: CommandOptions, config: ProjectConfig): Promise<any> {
+  async run(options: CommandOptions, config: ProjectConfig): Promise<any> {
     // Defer dependency loading until this specific command is run
-    const build = require('../build/build').build;
+    let build = require('../build/build').build;
+    const hasCLIArgumentsPassed = this.args.some(arg => {
+      return typeof options[arg.name] !== 'undefined';
```

Comment:
I prefer `typeof !== 'undefined'` vs `!== undefined`, you never know if any of your dependencies are messing with `undefined` so why risk it.